tools/vms-generator/utils/utils.go

```diff
@@ -1051,3 +1052,14 @@ func GetVMIMacvtap() *v1.VirtualMachineInstance {
 	vmi.Spec.Domain.Devices.Interfaces = []v1.Interface{{Name: macvtapNetworkName, InterfaceBindingMethod: v1.InterfaceBindingMethod{Macvtap: macvtap}}}
 	return vmi
 }
+
+func GetVMIARM() *v1.VirtualMachineInstance {
+	vmi := getBaseVMI(VmiARM)
```

Comment:
Would you mind adding a comment here to note why Arm64 VMs need 256Mi memory, something like
```
// The minimum memory for UEFI boot on Arm64 is 256Mi
```

Reply:
Ok, sure